integration/oasis/osrmconnector/osrmhttpclient.go

```diff
@@ -84,6 +105,12 @@ func (oc *osrmHTTPClient) response(m *message) {
 	if m.req.t == OSRMRoute {
 		routeResp.Err = json.NewDecoder(m.resp.Body).Decode(&routeResp.Resp)
 		m.req.routeRespC <- routeResp
+	} else if m.req.t == OSRMTable {
+		tableResp.Err = json.NewDecoder(m.resp.Body).Decode(&tableResp.Resp)
+		m.req.tableRespC <- tableResp
+	} else {
+		glog.Error("osrmHTTPClient Only support OSRMRoute and OSRMTable request for now.")
```

Comment:
same as before, use `glog.Fatal` instead of `glog.Error` + `panic` will be better.